cmd/create/idp/cmd.go

```diff
@@ -258,12 +266,28 @@ func init() {
 		"OpenID: List of scopes to request, in addition to the 'openid' scope, during the authorization token request.\n",
 	)
 
+	// HTPasswd
+	flags.StringVar(
+		&args.htpasswdUsername,
+		"username",
+		"",
+		"HTPasswd: Username to log into the cluster console with.\n",
+	)
+	flags.StringVar(
+		&args.htpasswdPassword,
+		"password",
+		"",
+		"HTPasswd: Password for provided username, to log into the cluster console with.\n",
+	)
+
 	interactive.AddFlag(flags)
+	reporter = rprtr.CreateReporterOrExit()
+	logger = logging.CreateLoggerOrExit(reporter)
 }
 
 func run(cmd *cobra.Command, _ []string) {
-	reporter := rprtr.CreateReporterOrExit()
-	logger := logging.CreateLoggerOrExit(reporter)
+	reporter = rprtr.CreateReporterOrExit()
+	logger = logging.CreateLoggerOrExit(reporter)
```

Comment:
Do we need lines 289-290?
if we already initialized `reporter` and `logger` in lines 284-285